flyway-community-db-support/flyway-database-clickhouse/src/main/java/org/flywaydb/community/database/clickhouse/ClickHouseDatabase.java

```diff
@@ -93,7 +100,11 @@ public String getRawCreateScript(Table table, boolean baseline) {
         String engine;
 
         if (isClustered) {
-            engine = "ReplicatedMergeTree('/clickhouse/tables/{shard}/{database}/{table}', '{replica}')";
+            if (hasCustomZookeeperPath) {
+                engine = "ReplicatedMergeTree('" + zookeeperPath + "', '{replica}')";
```

Comment:
'/clickhouse/tables/{shard}/{database}/{table}' this should be the default value for the new configuration param. I would suggest to set this value to ClickHouseConfigurationExtension::zookeeperPath as initial value so we keep current line for all cases and remove all redundant conditions.